packages/grafana-ui/src/components/NodeGraph/ViewControls.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import { Button } from '../Button';
+import { stylesFactory, useTheme } from '../../themes';
+import { GrafanaTheme } from '@grafana/data';
+import { css } from 'emotion';
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => ({
+  minusButton: css`
+    margin-left: 2px;
+  `,
+
+  scale: css`
+    font-size: 10px;
+    color: ${theme.colors.textFaint};
+  `,
+
+  scrollHelp: css`
+    font-size: 8px;
+    color: ${theme.colors.textFaint};
+  `,
```

Comment:
maybe use theme values for font-size?
```suggestion
  scale: css`
    font-size: ${theme.typography.size.sm};
    color: ${theme.colors.textFaint};
  `,

  scrollHelp: css`
    font-size: ${theme.typography.size.xs};
    color: ${theme.colors.textFaint};
  `,
```
Before:
![Screenshot 2021-01-19 at 10 22 17](https://user-images.githubusercontent.com/1170767/105021387-63944400-5a40-11eb-9661-ca1a52c97e64.png)
After:
![Screenshot 2021-01-19 at 10 21 31](https://user-images.githubusercontent.com/1170767/105021418-6b53e880-5a40-11eb-8c28-64328d1374fc.png)